pom.xml

```diff
@@ -49,7 +49,7 @@
     <argLine>-Xmx2g -Xms2g -Djava.awt.headless=true -XX:+HeapDumpOnOutOfMemoryError -XX:+TieredCompilation -XX:TieredStopAtLevel=1</argLine>
   </properties>
 
-  <scm>
+  <scm child.scm.connection.inherit.append.path="false" child.scm.developerConnection.inherit.append.path="false" child.scm.url.inherit.append.path="false">
```

Comment:
why this? is it related to this PR?

Reply:
Sort of—I had problems testing this incremental build in PCT, and was hoping that fixing the deployed `scm`s might fix it.

Reply:
Regardless of timing, this change is just matching the current recommendations: https://github.com/jenkinsci/archetypes/pull/360